source/extensions/common/wasm/wasm.h

```diff
@@ -697,7 +697,7 @@ class Wasm : public Envoy::Server::Wasm,
   // TODO(jplevyak): replace the use of Stats::StatNameSet with something more efficient.
   // By having a separate StatNameSet per Wasm we are duplicating all the strings but
   // avoiding locks. Consider lock-free hash tables or pre-registering stats.
-  Stats::StatNameSet stat_name_set_;
+  Stats::StatNameSetPtr stat_name_set_;
```

Comment:
@jplevyak PTAL, since it's going to conflict with #229.

Reply:
NP, I'll fix it in my merge.

Reply:
Thanks @jplevyak!